### PR TITLE
Testing part 2

### PR DESCRIPTION
**Pull request by erickma1**
In this milestone, my partner and I created tests for;

> A function to test editing a task
> A function for updating completed tasks
> A function to "clear all completed tasks"

### CODE REVIEW

File: src/modules/listfuns.test.js [hunk 1]
```diff
@@ -60,4 +61,52 @@ describe('ToDo-List: All CRUD Functions', () => {
       expect(liElements).toHaveLength(4);
     });
   });
+
```
Comment (1GORDON) [inline src/modules/listfuns.test.js:64]: Great job team 🤝

- [ ] As stated in the project requirements, kindly add tests for editing a task description.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @erickma1, and @Yonasle ✋

Good job so far!

## Highlights!
- Tested the update tasks function ✔️
- Tested Clear-All-Completed function ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by erickma1** [comment]
Add tests for editing a task description

File: src/modules/listfuns.test.js (hunk 1 above)
Comment (peterrobert) [reply src/modules/listfuns.test.js:64]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @erickma1, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️
[ ] Great work making the change requested by the previous reviewer. All the tests are passing and the Edit task test is now created✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**